lib/developer_portal/lib/liquid/tags/active_docs.rb

```diff
@@ -16,19 +16,19 @@ def initialize(tag_name, params, tokens)
         super
         if params =~ Syntax
           @version  = $1.strip
-          @services = $2.nil? ? [] : $2.split(",").map(&:strip)
+          @system_names = $2.nil? ? [] : $2.split(",").map(&:strip)
```

Comment:
I personally advocate for:
```suggestion
          @system_names = ($2 || '').split(",").map(&:strip)
```
Because the only reason is doing the `nil` check is to don't break when it does `split` because it expects a String. So this way you send a String as it expects and if `$2` contains `nil`, you end up returning `[]` as well 😄